etcd panic on invalid memory address when RecvAppendReq

```
2015/04/17 19:27:40 etcdserver: heartbeat = 100ms
2015/04/17 19:27:40 etcdserver: election = 1000ms
2015/04/17 19:27:40 etcdserver: snapshot count = 10000
2015/04/17 19:27:40 etcdserver: advertise client URLs = http://10.240.172.103:2379
2015/04/17 19:27:40 etcdserver: loaded cluster information from store: etcd-0=http://10.240.190.54:2380,etcd-1=http://10.240.172.103:2380,etcd-2=http://10.240.21.79:2380
2015/04/17 19:27:40 etcdserver: restart member a9b0ebadefc94b6f in cluster 7a25f25b7fd507b0 at commit index 4166825
2015/04/17 19:27:40 INFO: raft: a9b0ebadefc94b6f became follower at term 232
2015/04/17 19:27:40 INFO: raft: newRaft a9b0ebadefc94b6f [peers: [4f8b3f8e6b7b,8d448318f55e9d1d,a9b0ebadefc94b6f], term: 232, commit: 4166825, applied: 4145644, lastindex: 4167097, lastterm: 232]
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x6235f5]

goroutine 38 [running]:
github.com/coreos/etcd/etcdserver/stats.(*statsQueue).Insert(0x0, 0xc20f686540)
        /Users/unihorn/gopath/src/github.com/coreos/etcd/etcdserver/stats/queue.go:63 +0x1f5
github.com/coreos/etcd/etcdserver/stats.(*ServerStats).RecvAppendReq(0xc20f576480, 0xc20b8988e0, 0xc, 0x39eb4)
        /Users/unihorn/gopath/src/github.com/coreos/etcd/etcdserver/stats/server.go:115 +0x16a
github.com/coreos/etcd/etcdserver.(*EtcdServer).Process(0xc2080b67e0, 0x7f4ea9c46ef8, 0xc208032348, 0x3, 0xa9b0ebadefc94b6f, 0x4f8b3f8e6b7b, 0xeb, 0xe8, 0x3f959d, 0xc2177d0000, ...)
        /Users/unihorn/gopath/src/github.com/coreos/etcd/etcdserver/server.go:335 +0x302
github.com/coreos/etcd/rafthttp.func·002()
        /Users/unihorn/gopath/src/github.com/coreos/etcd/rafthttp/peer.go:175 +0x803
created by github.com/coreos/etcd/rafthttp.startPeer
        /Users/unihorn/gopath/src/github.com/coreos/etcd/rafthttp/peer.go:195 +0x970
```

--- a/etcdserver/server.go
+++ b/etcdserver/server.go
@@ -248,6 +248,7 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 		Name: cfg.Name,
 		ID:   id.String(),
 	}
+	sstats.Initialize()
 	lstats := stats.NewLeaderStats(id.String())
 
 	srv := &EtcdServer{
@@ -297,7 +298,6 @@ func (s *EtcdServer) start() {
 	s.w = wait.New()
 	s.done = make(chan struct{})
 	s.stop = make(chan struct{})
-	s.stats.Initialize()
 	// TODO: if this is an empty log, writes all peer infos
 	// into the first entry
 	go s.run()

